### PR TITLE
Replace `Buffer` with `b4a`

### PR DESCRIPTION
**Pull request by kasperisager**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,51 +8,48 @@ const STATEBYTES = sodium.crypto_secretstream_xchacha20poly1305_STATEBYTES
 const HEADERBYTES = sodium.crypto_secretstream_xchacha20poly1305_HEADERBYTES
 const KEYBYTES = sodium.crypto_secretstream_xchacha20poly1305_KEYBYTES
 
-if (!TAG_FINAL) throw new Error('JavaScript sodium version needs to support crypto_secretstream_xchacha20poly')
```
Comment (kasperisager) [inline index.js:10]: This was causing a panic in my browser bundle even though the module wasn't actually used, just imported.

Comment (mafintosh) [reply index.js:10]: Can we move this check into the constructor maybe?

File: package.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
   "description": "Wraps libsodiums secretstream in a higher level abstraction",
   "main": "index.js",
   "dependencies": {
+    "b4a": "^1.1.1",
     "sodium-universal": "^3.0.4"
   },
-  "devDependencies": {},
```
Comment (mafintosh) [inline package.json:9]: keep this, i like to have the empty map there so npm adds the deps in the same place (we should prob add tests also haha)

Comment (kasperisager) [reply package.json:9]: It was npm itself that nuked it so it might be a bit of an uphill battle 🙈

Comment (mafintosh) [reply package.json:9]: Can you just add standard for now?

**Comment by mafintosh** [comment]
1.0.2

**Pull request merged**